mop/settings.py

```diff
@@ -275,6 +275,7 @@
     'TAP_priority': ('TAP_priority', 'TAP Priority'),
     'TAP_priority_longtE': ('TAP_priority_longtE', 'TAP Priority Long-tE'),
     'lc_model': ('lc_model', 'Model'),
+    'pylima_model': ('pylima_model', 'pyLIMA Model'),
```

Comment:
This new data type is really pre-formatted HTML isn't it?  I am wondering if it is worth generalising this so that we can use the same type for other pre-build HTML plots, such as those built by Plotly for other tasks.  If so then please generalise the name.

Reply:
This is actually an identifier. The code looks for this type of identifier in order to display the correct type dataproduct file in the display (in this case, a pyLIMA model). The generic type it belong to is dataproduct so an html file can be easily uploaded as most dataproduct types.